README.md

```diff
@@ -106,7 +100,7 @@ Usage in an HTML (or alike) file:
 
 #### Require Syntax <!-- omit in toc -->
```

Comment:
Also here, @bidoubiwa: from "Require syntax" to "CommonJS modules"?

It's possible I'm just old, but whenever I read this "Require syntax" I think about [requireJS](https://requirejs.org/)/AMD modules instead of Node-style CommonJS modules.

Reply:
Based on our call:

```suggestion
#### `require` syntax <!-- omit in toc -->
```